python/ray/data/tests/test_arrow_block.py

```diff
@@ -1,15 +1,251 @@
+import os
+import sys
 import types
+from tempfile import TemporaryDirectory
+from typing import Union
 
 import numpy as np
 import pyarrow as pa
 import pytest
+from pyarrow import parquet as pq
 
 import ray
 from ray._private.test_utils import run_string_as_driver
+from ray.air.util.tensor_extensions.arrow import ArrowTensorArray
+from ray.data import DataContext
 from ray.data._internal.arrow_block import ArrowBlockAccessor
+from ray.data._internal.arrow_ops.transform_pyarrow import combine_chunked_array
+from ray.data._internal.util import GiB, MiB
 from ray.data.extensions.object_extension import _object_extension_type_allowed
 
 
+@pytest.fixture(scope="module")
+def parquet_dataset_single_column_gt_2gb():
+    chunk_size = 256 * MiB
+    num_chunks = 10
```

Comment:
this should be MIN_NUM_CHUNKS_TO_TRIGGER_COMBINE_CHUNKS?
If so, refer to the constant instead of defining a magical number here

Reply:
This actually has nothing to do with it -- it's purpose to get us over 2Gb (check line 27)